README.md

```diff
@@ -242,3 +242,28 @@ The stop_delays view looks like this:
 ### Demo Project
 
 See the [gtfsrdb-delay-demo](https://github.com/CUTR-at-USF/gtfsrdb-delay-demo) project for a sample web application that visualizes these delays.
+
+### Scripts
+
+The scripts/ directory is provided as a "quick start" method for using GTFSrDB. A database server is still a prerequisite, and the scripts assume you're using PostgreSQL.
+
+To get started with the scripts, first make sure you have a good working Python environment. This involves setting up a virtual environment, then installing the required libraries. A quick way to do this is as follows:
+1. Ensure you are in the root directory of gtfsrdb on your PC.
+2. Create a Python virtual environment: `python3 -m venv venv`
+3. Install the required libraries: `pip3 install -r requirements.txt`
+
+Once your environment is configured, edit `env.sh` to ensure the `DB_HOST`, `DB_USER`, and `DB_PASS` are correct for your application. Also note that the `run.sh` script will create a database with the name specified by `DB`, so you may also alter this if you wish.
+
+Once your `env.sh` is configured, you can use `run.sh`. There are two ways to use this script:
+1. Specify all of your options on the command line. For example, if you were using the Tampa GTFS-RT feed, you would use something like the following:
+`./run.sh tampa -t http://api.tampa.onebusaway.org:8088/trip-updates`. This will create a PostgreSQL database using the `DB` from `env.sh`, create a table in that database called `tampa`, and populate the database's `trip_updates` table with the data received from the GTFS-RT feed. You can also subscribe to other feeds (alerts or position updates) or apply other gtfsrdb options just as you would if you ran `gtfsrdb.py` from the command line directly.
+2. Create a text file with the feeds you want to subscribe to and use the `-I` option in the `run.sh` script. For example, you could create a file for the StanRTA GTFS-RT fields called `stanrta.txt` and populate it like so:
+```
+-p https://max.availtec.com/InfoPoint/GTFS-Realtime.ashx?Type=VehiclePosition
+-t https://stanrta.rideralerts.com/InfoPoint/GTFS-Realtime.ashx?Type=TripUpdate
+-a https://stanrta.rideralerts.com/InfoPoint/GTFS-Realtime.ashx?Type=Alert
+```
+Then use the following to invoke the `run.sh` script:
+`./run.sh stanrta -I stanrta.txt`. Once again, you can add other `gtfsrdb.py` options, so if you wanted to use verbose mode, for example, you would run: `./run.sh stanrta -I stanrta.txt -v`.
+
+The `rmdb.sh` script is a quick way to clean up your PostgreSQL server if you need to remove databases that you no longer need. For example, if you needed to get rid of the `stanrta` database, you could run `./rmdb.py stanrta`.
```

Comment:
Same here: We could add a flag to drop the database before storing something which comes in handy during development